src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_commitfilelist.yaml

```diff
@@ -0,0 +1,113 @@
+post:
+  summary: Commit package changes from a filelist.
+  description: |
+    Create a new commit using defined files already uploaded to the repository.
+    This can be done via `PUT` using `rev=repository` which avoids a commit.
```

Comment:
I find this line confusing, even if it copied from /apidocs, because there is only a mention to the action (`PUT`) but not to the path.

Could we clarify what API endpoint (action with path) are we referring to here?

It would also be fine for me if this line is removed.